pkg/endpoint/policy.go

```diff
@@ -498,6 +499,230 @@ func (e *Endpoint) IngressOrEgressIsEnforced() bool {
 		e.Opts.IsEnabled(OptionEgressPolicy)
 }
 
+func (e *Endpoint) GetPortNumber(port string) (uint16, error) {
```

Comment:
This feels pretty low in the callstack to be doing string validation.

Reply:
It's not string validation. We need to support kubernetes ports names in the policy. A policy can have "80" or "my-port". The container / endpoint running will have the association of that name to a port where the service is running

Reply:
Ah, okay. So this will be extended to handle named ports. Fair enough.